code/game/atoms_movable.dm

```diff
@@ -134,7 +134,7 @@
 		target = get_step_multiz(source, direction)
 		if(!target)
 			return FALSE
-	return !(movement_type & FLYING) && has_gravity(source) && !throwing
+	return !(movement_type & FLYING) && has_gravity(source) && !throwing && !isAI(src)
```

Comment:
Doesn't this just affect the ai core mob and not the ai camera mob?

Reply:
You'd think that, but when you're in an AI core looking through the camera, you're still just a `/mob/living/silicon/ai` and not `/mob/camera/ai_eye`.

The AI camera eye actually moves through /mob/living/silicon/ai/zMove() which moves their eyeobj instead.

Reply:
why not make `movement_type & FLYING` to `/mob/living/silicon/ai`?

Reply:
Because the AI core is not flying and would benefit from any checks that involve the AI core as a mob and whether it was flying or not.